packages/core/src/components/JsonBlock/JsonBlock.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Button, Card, CardContent } from '@mui/material';
+import { VerifiableCredential } from '@vckit/core-types';
+
+const JsonBlock = ({ credential }: { credential: VerifiableCredential }) => {
+  /**
+   * handle click on download button
+   */
+  const handleClickDownloadVC = async () => {
```

Comment:
Do you think we should write this in function for reusable?

Reply:
Yes, I agree with you, but currently, it's only used in this place. So I don't want to make the core package more complex by splitting this function to another place. When another component uses this function, then I will move it to the utils file to facilitate easier reuse. Thank you for your feedback.